soda/core/soda/execution/schema_check.py

```diff
@@ -61,15 +59,10 @@ def evaluate(self, metrics: Dict[str, Metric], historic_values: Dict[str, object
 
         self.measured_schema: List[Dict[str, str]] = metrics.get(KEY_SCHEMA_MEASURED).value
 
-        if KEY_SCHEMA_PREVIOUS in historic_values and historic_values.get(KEY_SCHEMA_PREVIOUS).get("measurements").get(
-            "results"
-        ):
-            logging.info("No previous schema checks available, skipping schema check")
-            self.skipped = True
-
         schema_previous_measurement = (
             historic_values.get(KEY_SCHEMA_PREVIOUS).get("measurements").get("results")[0].get("value")
```

Comment:
@vijaykiran should we point to the first-ever created schema reference or the latest schema reference in order to run schema check. In other words, should I keep 0 or change it to -1 :D

Reply:
Good point - I think `-1` is better too.